funnel/forms/venue.py

```diff
@@ -28,6 +28,9 @@ class VenueForm(forms.Form):
     country = forms.SelectField(__("Country"),
         validators=[forms.validators.Optional(), forms.validators.Length(max=2)],
         choices=country_codes, default="IN")
+    seq = forms.IntegerField(__("Sequence"),
+        description=__("The sequence of the venue on the listing"),
+        validators=[forms.validators.DataRequired(__("Please specify the sequence order"))])
```

Comment:
This should not be in the form. We want drag and drop reordering.

Reply:
@jace Do we add the drag and drop as part of room settings on the schedule edit page.


<img width="150" alt="screen shot 2019-02-07 at 2 36 28 pm" src="https://user-images.githubusercontent.com/11267213/52400680-d28f6100-2ae5-11e9-9d35-989807d1e544.png">